x-pack/plugins/maps/public/classes/layers/blended_vector_layer/blended_vector_layer.ts

```diff
@@ -305,6 +300,7 @@ export class BlendedVectorLayer extends VectorLayer implements IVectorLayer {
       prevDataRequest: this.getDataRequest(dataRequestId),
       nextMeta: searchFilters,
       extentAware: source.isFilterByMapBounds(),
+      getUpdateDueToTimeslice: this._getUpdateDueToTimesliceFromSourceRequestMeta,
```

Comment:
this needs to be wrapped in a lambda. The `this`-binding is not preserved for methods.

```
vector_layer.tsx:992 Uncaught (in promise) TypeError: Cannot read property 'getSourceDataRequest' of undefined
    at _getUpdateDueToTimesliceFromSourceRequestMeta (vector_layer.tsx:992)
    at canSkipSourceUpdate (can_skip_fetch.ts:75)
    at async BlendedVectorLayer.syncData (blended_vector_layer.ts:225)
    at async data_request_actions.ts:114
    at async Promise.all (:5601/ofh/app/maps/index 1)
    at async data_request_actions.ts:91
    at async map_actions.ts:231
```